source/elements/oneDPL/source/parallel_api/parallel_range_api.rst

```diff
@@ -458,6 +472,18 @@ Copying Mutating Operations
         transform (ExecutionPolicy&& pol, R1&& r1, R2&& r2, OutR&& result, Fn binary_op,
                    Proj1 proj1 = {}, Proj2 proj2 = {});
 
+    // unique_copy
+    template <typename ExecutionPolicy, std::ranges::random_access_range R,
+              std::ranges::random_access_range OutR, typename Proj = std::identity,
+              std::indirect_equivalence_relation< std::projected<std::ranges::iterator_t<R>, Proj> >
```

Comment:
I believe this diverged from our normal formatting with extra spaces.

```suggestion
              std::indirect_equivalence_relation<std::projected<std::ranges::iterator_t<R>, Proj>>
```

Reply:
There were some cases with additional spaces, for example `std::indirect_result_t<Fn&, std::projected<std::ranges::iterator_t<R>, Proj>> >`, but it differs from algorithm to algorithm. I've removed them here.